.changeset/eleven-wasps-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
```

Comment:
```suggestion
'@sumup/circuit-ui': minor
```

Didn't we want to make this specific change a minor? The dropping of browser support will be the major change 🙂

Reply:
🤦‍♂️ 

Yes. I meant to manually edit the file but forgot. Thank you for spotting it!